src/app/tasks/task-ilo-alignment/task-ilo-alignment-rater/task-ilo-alignment-rater.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit, Inject } from '@angular/core';
+import { outcomeService } from 'src/app/ajs-upgraded-providers';
+
+@Component({
+  selector: 'task-ilo-alignment-rater',
+  templateUrl: 'task-ilo-alignment-rater.component.html',
+  styleUrls: ['task-ilo-alignment-rater.component.scss'],
+}) 
+export class TaskIloAlignmentRaterComponent implements OnInit {
+  @Input() readonly: boolean = true;
+  // Alignment object
```

Comment:
These comments are redundant, in the old coffee file they attempted to explain something about the property, here they mostly repeat the name of the property